src/ui/components/Timeline/index.tsx

```diff
@@ -101,7 +103,7 @@ export default function Timeline() {
   return (
     <>
       <FocusModePopout />
-      <div className="timeline">
+      <div className={`timeline ${showProtocolTimeline ? "show-protocol-timeline" : ""}`}>
```

Comment:
I prefer how `classNames` would read here, but don't feel strongly about it.

Reply:
me too - but lazy